### PR TITLE
Add Gravity Docs Readme

### PR DESCRIPTION
**Pull request by benarent**
Provides the same instructions as Teleport Docs, makes it clearer for anyone else looking to edit / preview these docs.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Gravity Docs
+
+Gravity docs are built using [mkdocs](http://www.mkdocs.org/) and hosted
+as a bunch of static files in S3. Located Here https://gravitational.com/gravity/docs/overview/ 
+
+Look at `build.sh` script to see how it works.
+
+## To Publish New Version
```
Comment (a-palchikov) [inline docs/README.md]: nit: `Publishing New Version` to keep titles consistent

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Gravity Docs
+
+Gravity docs are built using [mkdocs](http://www.mkdocs.org/) and hosted
+as a bunch of static files in S3. Located Here https://gravitational.com/gravity/docs/overview/
```
Comment (a-palchikov) [inline docs/README.md]: ... and hosted as static files on S3. Located **h**ere ...

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Gravity docs are built using [mkdocs](http://www.mkdocs.org/) and hosted
+as a bunch of static files in S3. Located Here https://gravitational.com/gravity/docs/overview/ 
+
+Look at `build.sh` script to see how it works.
+
+## To Publish New Version
+
+* Update [build.sh](build.sh).
+* Update theme/base.html to add a new version to the Javascript arrray of versions
```
Comment (a-palchikov) [inline docs/README.md]: arr~~r~~ay

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Create a new YAML file, like `5.5.1.yaml` if you are releasing version 5.5.1
+
+## Deploying
+
+Gravity docs are published using a private `web` repository.
```
Comment (a-palchikov) [inline docs/README.md]: a -> the private `web` repository?

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+See `web/README.md` for more info.
+
+## Running Locally
+
+We recommend using Docker to run and build the docs. 
+
+`make run-docs` will run the docs and setup a [livereload server](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en) for easy previewing of changes.
```
Comment (a-palchikov) [inline docs/README.md]: What does `will run the docs` mean - will start a local server to view the documentation?
I also failed to find the `run-docs` target in Makefile (there's just `run`).

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+We recommend using Docker to run and build the docs. 
+
+`make run-docs` will run the docs and setup a [livereload server](https://chrome.google.com/webstore/detail/livereload/jnihajbhpnppcggbcgedagnkighmdlei?hl=en) for easy previewing of changes.  
+
+`make docs` will build the docs, so they are ready to ship to production. 
+
+
+## Tools used to build the Docs
+
+Gravity Docs are made with MkDocs and a few markdown extensions, First time users will need to install MkDocs https://www.mkdocs.org/#installation.
```
Comment (a-palchikov) [inline docs/README.md]: nit: Docs -> docs
... will need to install [MkDocs](https://www.mkdocs.org/#installation).

File: docs/README.md (hunk 5 above)
Comment (benarent) [reply docs/README.md]: Thanks for the review @a-palchikov. 

I've been running it from the root directory but maybe I should add the command to the /docs/ directory.

Comment (benarent) [reply docs/README.md]: Changed `make run-docs` to `make run` for clarity.

**Pull request merged**